tools/sz_repo_cli/lib/src/common/src/app_store_connect_utils.dart

```diff
@@ -46,7 +50,9 @@ Future<void> _keychainInitialize() async {
 Future<void> _fetchSigningFiles({
   required AppleSigningConfig config,
 }) async {
-  const bundleId = 'de.codingbrain.sharezone';
+  // Even though our actual bundle ID has also the suffix ".app", we need to
+  // use here a different bundle ID
+  const bundleId = 'de.codingbrain.sharezone.app';
```

Comment:
I don't understand what the comment is saying and why exactly we need a different bundle id

Reply:
Ah, I forgot to remove the comment.